### DISCUSSION TITLE
Messages section under utilities

### DISCUSSION THREAD

**Discussion by lukeyouell** [craftcms/cms#8501] [Ideas]
I think it would be handy to have a messages section under utilities where plugin developers could place 'broadcasts' for craft installations that use their plugin(s)

For example I've recently changed my plugin package names, switching from `craft3-` to `craft-` but I have no way of telling this to people who have them installed.

I was thinking something as simple as setting a message from id.craftcms.com which is then visible from that section in the utilities if they have that plugin installed.

![image 1](https://user-images.githubusercontent.com/24975058/36066212-15244572-0e9e-11e8-9f57-af621bf42671.png)

**Comment by jalendport** [comment]
This would be very awesome 👍
